pv.js

```diff
@@ -63,7 +66,13 @@ class Pv {
             this.pvLine = null;
         }
     }
-
+    postPvToChatFree(errline) {
+        const stop = this.STOPRE.exec(errline);
+        if (stop && this.pvLine) {
+            const body = this.getPvChat(stop);
+            this.game.sendChat(body, "malkovich");
```

Comment:
The `sendChat` function has 3 parameters, body, move number and type (malkovich, or the default, discussion). You are sending the string malkovich as move number now, ogs does not understand this.

Either send no move number, or be nice and send the current move as move number. Either way make "malkovich" the third parameter if you intend to send this to malkoch log. if not you could just call `this.game.sendChat(body, this.game.state.moves.length + 1)` instead.

The other functions have  +1 on the moves to make the pv land on the correct move. We probably want that here too.